pom.xml

```diff
@@ -11,12 +11,12 @@
 
   <scm>
     <developerConnection>scm:git:https://github.com/usdot-jpo-ode/jpo-ode.git</developerConnection>
-    <tag>jpo-ode-3.0.0-SNAPSHOT</tag>
+    <tag>jpo-ode-4.0.0-SNAPSHOT</tag>
   </scm>
 
   <groupId>usdot.jpo.ode</groupId>
   <artifactId>jpo-ode</artifactId>
-  <version>3.0.0-SNAPSHOT</version>
+  <version>4.0.0-SNAPSHOT</version>
```

Comment:
Defined `project.ode.version` property in pom.xml

Reply:
Reverted this change since SONAR warnings indicated the version should be a constant, not an expression: https://github.com/CDOT-CV/jpo-ode/actions/runs/12125880910/job/33806788964